### PR TITLE
[zh-cn] Resync pod security standards page

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/zh-cn/docs/concepts/security/pod-security-standards.md [hunk 1]
```diff
@@ -97,151 +98,74 @@ fail validation.
...
-         <a href="/zh-cn/docs/tasks/configure-pod-container/create-hostprocess-pod">HostProcess 容器</a> 的能力，
-         这使得对 Windows 节点的特权访问成为可能。 
-         基线策略中对宿主的特权访问是被禁止的。 
-         HostProcess Pod 是 Kubernetes <strong>v1.22</strong> 版本的
-          <strong>alpha</strong> 特性。</p>
-				<p><strong>限制的字段</strong></p>
+      			<td style="white-space: nowrap">HostProcess</td>
+      			<td>
+				<p><!--Windows pods offer the ability to run <a href="/docs/tasks/configure-pod-container/create-hostprocess-pod">HostProcess containers</a> which enables privileged access to the Windows node. Privileged access to the host is disallowed in the baseline policy. -->
+                                Windows Pod 提供了运行 <a href="/zh-cn/docs/tasks/configure-pod-container/create-hostprocess-pod">HostProcess 容器</a> 的能力，这使得对 Windows 节点的特权访问成为可能。 Baseline 策略中禁止对宿主的特权访问。{{< feature-state for_k8s_version="v1.23" state="beta" >}}
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/security/pod-security-standards.md:106]:
```suggestion
                                Windows Pod 提供了运行 <a href="/zh-cn/docs/tasks/configure-pod-container/create-hostprocess-pod">HostProcess 容器</a> 的能力，这使得对 Windows 节点的特权访问成为可能。Baseline 策略中禁止对宿主的特权访问。{{< feature-state for_k8s_version="v1.23" state="beta" >}}
```

File: content/zh-cn/docs/concepts/security/pod-security-standards.md [hunk 2]
```diff
@@ -605,210 +382,107 @@ fail validation.
...
+				<p><strong><!--Allowed Values-->准许的取值</strong></p>
 				<ul>
 					<li><code>RuntimeDefault</code></li>
 					<li><code>Localhost</code></li>
 				</ul>
 				<small>
-					如果 Pod 级别的 <code>spec.securityContext.seccompProfile.type</code> 
-          已设置得当，容器级别的安全上下文字段可以为 未定义/<code>nil</code>。
-          反过来说，如果 _所有的_ 容器级别的安全上下文字段已设置，则 Pod 级别的字段可为 未定义/<code>nil</code>。 
+					<!--The container fields may be undefined/<code>nil</code> if the pod-level <code>spec.securityContext.seccompProfile.type</code> field is set appropriately.  Conversely, the pod-level field may be undefined/<code>nil</code> if _all_ container- level fields are set.-->如果 Pod 级别的 <code>spec.securityContext.seccompProfile.type</code> 已设置得当，容器级别的安全上下文字段可以为 未定义/<code>nil</code>。 反过来说，如果 <bold>所有的</bold> 容器级别的安全上下文字段已设置，则 Pod 级别的字段可为 未定义/<code>nil</code>。
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/security/pod-security-standards.md:456]:
```suggestion
					<!--The container fields may be undefined/<code>nil</code> if the pod-level <code>spec.securityContext.seccompProfile.type</code> field is set appropriately.  Conversely, the pod-level field may be undefined/<code>nil</code> if _all_ container- level fields are set.-->如果 Pod 级别的 <code>spec.securityContext.seccompProfile.type</code> 已设置得当，容器级别的安全上下文字段可以为 未定义/<code>nil</code>。反过来说，如果 <bold>所有的</bold> 容器级别的安全上下文字段已设置，则 Pod 级别的字段可为 未定义/<code>nil</code>。
```

File: content/zh-cn/docs/concepts/security/pod-security-standards.md [hunk 3]
```diff
@@ -953,7 +633,7 @@ There is not currently an API standard that controls whether a Pod is considered
 not. Sandbox Pods may be identified by the use of a sandboxed runtime (such as gVisor or Kata
 Containers), but there is no standard definition of what a sandboxed runtime is.
 -->
-### 沙箱（Sandboxed） Pod 怎么处理？
+### 沙箱（Sandboxed）Pod 怎么处理？
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/security/pod-security-standards.md:636]:
```suggestion
### 沙箱（Sandboxed）Pod 怎么处理？   {#what-about-sandboxed-pods}
```

File: content/zh-cn/docs/concepts/security/pod-security-standards.md [hunk 4]
```diff
@@ -97,151 +98,74 @@ fail validation.
 			<td>控制（Control）</td>
 			<td>策略（Policy）</td>
 		</tr>
-    <tr>
+    		<tr>
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/security/pod-security-standards.md:101]: The original file is already mixed spaces and tabs :(

File: content/zh-cn/docs/concepts/security/pod-security-standards.md [hunk 5]
```diff
@@ -259,150 +183,74 @@ fail validation.
...
-					<li>Undefined/nil</li>
-					<li><code>runtime/default</code></li>
-					<li><code>localhost/*</code></li>
-				</ul>
-			</td> -->
 			<td>
-        <p>在受支持的主机上，默认使用 <code>runtime/default</code> AppArmor Profile。
-				基线策略应避免覆盖或者禁用默认策略，以及限制覆盖一些 Profile 集合的权限。</p>
-				<p><strong>限制的字段</strong></p>
+				<p><!--On supported hosts, the <code>runtime/default</code> AppArmor profile is applied by default. The baseline policy should prevent overriding or disabling the default AppArmor profile, or restrict overrides to an allowed set of profiles.-->在受支持的主机上，默认使用 <code>runtime/default</code> AppArmor 配置。Baseline 策略应避免覆盖或者禁用默认策略，以及限制覆盖一些 配置集合的权限。</p>
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/security/pod-security-standards.md:221]: Weird space.
```suggestion
				<p><!--On supported hosts, the <code>runtime/default</code> AppArmor profile is applied by default. The baseline policy should prevent overriding or disabling the default AppArmor profile, or restrict overrides to an allowed set of profiles.-->在受支持的主机上，默认使用 <code>runtime/default</code> AppArmor 配置。Baseline 策略应避免覆盖或者禁用默认策略，以及限制覆盖一些配置集合的权限。</p>
```

**Review by Sea-n — COMMENTED** [review]
/lgtm

File: content/zh-cn/docs/concepts/security/pod-security-standards.md (hunk 4 above)
Comment (tengqm) [reply content/zh-cn/docs/concepts/security/pod-security-standards.md:101]: Tabs should be removed. It warrants another PR.

**Pull request merged**